main/WebServer.cpp

```diff
@@ -4308,7 +4308,8 @@ namespace http {
               (dipswitchcode == "")
               )
               return;
-						sprintf(szTmp, "%03X", std::stoi(dipswitchcode, nullptr, 2));
+            char * pEnd;
+						sprintf(szTmp, "%03lX", std::strtol(dipswitchcode.c_str(), &pEnd, 2));
```

Comment:
This is base 2 number, not base 10. The switches have a row of 10 dipswitches for configuration. most people cant do a base 2 to base 10 or base 16 conversion by heart, so the input takes 10 value binary string,